test/e2e_node/runner/remote/run_remote.go

```diff
@@ -514,13 +526,37 @@ func createInstance(imageConfig *internalGCEImage) (string, error) {
 			},
 		},
 	}
-	i.Metadata = imageConfig.metadata
-	op, err := computeService.Instances.Insert(*project, *zone, i).Do()
-	if err != nil {
-		return "", err
+
+	for _, accelerator := range imageConfig.resources.Accelerators {
+		if i.GuestAccelerators == nil {
```

Comment:
nit: single letter non-index variable (especially `i`) in a for loop made me do a double-take, even though it's technically correct in this case